src/containers/TemplateBuilder/template/Actions/ActionConfig.tsx

```diff
@@ -129,7 +129,7 @@ const ActionConfig: React.FC<ActionConfigProps> = ({ templateAction, onClose })
           style={{ borderTopRightRadius: 8 }}
         >
           <a
-            href="https://github.com/openmsupply/conforma-server/wiki/List-of-Action-plugins"
+            href="https://github.com/msupply-foundation/conforma-server/wiki/List-of-Action-plugins"
```

Comment:
This was changed company-wide for our organisation. It does forward correctly, but I'm updating them as I find them.